db/migrate/20180315163832_add_hide_comment_field_to_resource_type.rb

```diff
@@ -0,0 +1,5 @@
+class AddHideCommentFieldToResourceType < ActiveRecord::Migration
+  def change
+    add_column :mybookings_resource_types, :hide_comment_field, :boolean
```

Comment:
We can set this field to FALSE by default.

Reply:
Ok.